zio-schema-avro/src/test/scala/zio/schema/codec/AvroSchemaCodecSpec.scala

```diff
@@ -81,7 +81,7 @@ object AvroSchemaCodecSpec extends ZIOSpecDefault {
             val expected =
               """[{"type":"record","name":"A","fields":[]},{"type":"record","name":"B","fields":[]},{"type":"record","name":"MyC","fields":[]},{"type":"record","name":"D","fields":[{"name":"s","type":"string"}]}]"""
             assert(result)(isRight(equalTo(expected)))
-          } @@ TestAspect.scala2Only,
+          } @@ TestAspect.scala2Only @@ TestAspect.ignore,
```

Comment:
looks like this test was created as a workaround for the incorrect Scala2 DeriveSchema behaviour for intermediate traits - so its OK to ignore/delete this test?